### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by srih4ri**

### CODE REVIEW

**Comment by srih4ri** [comment]
@srih4ri _a do it yourself.

**Comment by sudarsh** [comment]
Hey!

**Comment by srih4ri** [comment]
@sudarsh _a Look at this.

**Comment by srih4ri** [comment]
@sudarsh _a Hey, look at this.

**Comment by sudarsh** [comment]
@srih4ri  _a check this issue bitch!

**Comment by srih4ri** [comment]
@sudarsh _a take it back

File: README.markdown [hunk 1]
```diff
@@ -1,2 +1,4 @@
+To run the program: 
+```
 ./pixelart.py example/pythocat.jpg 3 3
---
```
Comment (srih4ri) [inline README.markdown:2]: @sudarsh _a test.

**Pull request closed without merging**